examples/mpc_linear_svm/launcher.py

```diff
@@ -23,8 +23,6 @@
 import logging
 import os
 
-from examples.multiprocess_launcher import MultiProcessLauncher
```

Comment:
This change was needed such that there is not needed to be sent when running this example on multiple machines.

We can also send this file, but we would also need to create the directory structure. (```examples``` ...)